Core/Core/DeveloperMenu/DeveloperMenuViewController.swift

```diff
@@ -189,17 +178,13 @@ extension DeveloperMenuViewController: UITableViewDataSource, UITableViewDelegat
                 UserDefaults.standard.synchronize()
                 //        case .logs:
             //            shareLogs()
-            case .enableAllExperimentalFeatures: break
+            case .experimentalFeatures:
+                env?.router.show(ExperimentalFeaturesViewController(), from: self)
```

Comment:
looks like `afterToggle` is only set in routes, but this doesn't use routes, it goes straight to the controller.

Reply:
Ah good catch.